uwblueprint/headsupguys
# Create Seeding Script

## Notion ticket link

[Database Seeding](https://www.notion.so/uwblueprintexecs/Task-Board-485f846ae00243599dddba2be9bcba82?p=18d5693d2f3145d390d4bca1b8bdc384)



## Implementation description
* created a script at `database/scripts/seed.ts` that populated collections with mock data
* moved model files into `database/models`
* renamed Module's `tool` to `toolID` and `slides` to `slideIDs`



## Steps to test
1. ensure `MONGODB_URL` and `MONGODB_DB` are set in `.env.local`
2. run `yarn seed --init` in the terminal (--init means it's the first time the script is being ran and won't try to drop collections)
3. run `yarn seed` for all subsequent seeding needs



## What should reviewers focus on?
* mock data matches schema


## Checklist
